pkg/kubelet/kubelet_linux.go

```diff
@@ -49,3 +56,52 @@ func (kl *Kubelet) cgroupVersionCheck() error {
 	}
 	return nil
 }
+
+func (kl *Kubelet) startHealthCheck() error {
+	if runtime.GOOS != "linux" {
+		return fmt.Errorf("systemd watchdog can only be used on linux")
```

Comment:
this will unconditionally report an error on Windows. Perhaps this check can be moved to the callsite and Windows simply be ignored